lib/processors/bundlers/flexChangesBundler.js

```diff
@@ -37,6 +37,9 @@ module.exports = function({resources, options: {pathPrefix, hasFlexBundleVersion
 		const variantManagementChanges = [];
 
 		changesContent.forEach(function(content) {
+			if (content.layer === "VENDOR") {
+				content.support.user = "SAP";
+			}
```

Comment:
Can't it be that layer is "VENDOR" and the user is not SAP? 
In other words: shouldn't the user only be set when there's no value for user yet?

Reply:
No, the VENDOR layer is only use by SAP. When the user is set, it have to override with SAP.

Reply:
Thank you for the clarification.